explorer/db/testutil.go

```diff
@@ -48,6 +50,7 @@ func CreateTestBlocks(t *testing.T, ctx context.Context, client *ent.Client, cou
 	var msg *ent.Msg
 	var blocks []ent.Block
 	for i := 0; i < count; i++ {
+		time.Sleep(10 * time.Millisecond)
```

Comment:
Why is this magic wait time needed? And how do you know 10ms is enough? Is there a better way to do this?

Reply:
Just to make sure the time stamp on the block entries in the test environment are different enough to sort by, I'll add some comments good catch